airflow/contrib/hooks/gcp_api_base_hook.py

```diff
@@ -159,6 +161,28 @@ def _get_field(self, f, default=None):
     def project_id(self):
         return self._get_field('project')
 
+    def convert_http_exception(func):
```

Comment:
I have a test in other PR.  This file is an area of deeper changes, so I preferred to limit the changes. https://github.com/apache/airflow/pull/4790